cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -195,6 +174,41 @@ body {
             font-size: unit( @base-font-size-px / @base-font-size-px, em );
             font-weight: normal;
         }
+
+        // Hide the ">" icon next to menu items on desktop…
+        &-link {
+            .cf-icon-svg {
+                display: none;
+            }
+        }
+
+        // …however, show the icons in featured links.
+        &_content-list__featured {
+            .cf-icon-svg {
+                display: inline-block;
+            }
+        }
+
+        // Featured menu content is link items with icons.
+        &-list__featured {
+
+            & .cf-icon-svg {
+                display: inline;
+                color: @pacific;
+                margin-right: 10px;
+            }
+        }
+
+        &-list__featured li {
+            // Align the icon and text.
+            display: flex;
+            align-items: baseline;
+
+            // Hide link bottom line from Design System link styles.
+            & .a-link_text {
+                border-bottom-width: 0;
```

Comment:
With the new markup this is the only flex item so add `flex-grow: 1` to ensure it stretches to the full width of the parent.